config/rbac/kustomization.yaml

```diff
@@ -7,8 +7,8 @@ resources:
 - service_account.yaml
 - role.yaml
 - role_binding.yaml
-- leader_election_role.yaml
-- leader_election_role_binding.yaml
+# - leader_election_role.yaml
+# - leader_election_role_binding.yaml
```

Comment:
can we delete these lines?

Reply:
I prefer to have this in comments, similar as in other places.